fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -238,6 +242,18 @@ public DataApiRequest(
 
         this.having = DruidHavingBuilder.buildHavings(this.havings);
 
+
+        if (
+                sorts != null &&
+                        !sorts.isEmpty() &&
+                        sorts.toLowerCase(Locale.ENGLISH).contains(DATE_TIME_STRING.toLowerCase(Locale.ENGLISH))
+                ) {
+            //Requested sort on dateTime - It has to be first field in the string
+            this.dateTimeSort = generateDateTimeSortColumn(sorts.substring(0, sorts.indexOf(",")));
+            //Separating dateTimeSort from the complete api sort value string
+            sorts = sorts.substring(sorts.indexOf(",") + 1, sorts.length());
```

Comment:
If `sorts` is a method parameter, it's generally a bad idea to modify method parameters. They should be treated as immutable.